<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrivateMessages</class>
 <widget class="QWidget" name="PrivateMessages">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>390</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>481</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="friendNameLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>friendName</string>
   </property>
  </widget>
  <widget class="QComboBox" name="friendCombobox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string extracomment="Pick a friend to chat with"/>
   </property>
  </widget>
  <widget class="QComboBox" name="serverCombobox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string extracomment="Pick a server tp view"/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>textEdit</tabstop>
  <tabstop>friendCombobox</tabstop>
  <tabstop>serverCombobox</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
